can't build wasm

I've tried building the wasm version following instructions from the readme, but CMake failed with this error:
```
CMake Error: Cannot determine link language for target "glslang.js_1_glslang_pre_js".
```
I'm on Mac and also tried ubuntu in docker. Both arm64 if that matters.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -41,7 +41,7 @@ if(POLICY CMP0054)
   cmake_policy(SET CMP0054 NEW)
 endif()
 
-project(glslang LANGUAGES CXX)
+project(glslang)
 
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 

